README.md

```diff
@@ -28,25 +28,35 @@ Documentation and guides are coming soon! In the mean time we suggest creating a
 
 ## To develop gRPC for ASP.NET Core
 
+Setting up local feed with unreleased Grpc.* packages (May not be necessary):
+```
+# We may depend on unreleased Grpc.* packages.
+# Run this script before building the project.
+./build/get-grpc.sh or ./build/get-grpc.ps1
+```
+
 Installing .NET Core SDK:
 ```
 # Run this script before building the project.
 ./build/get-dotnet.sh or ./build/get-dotnet.ps1
 ```
 
-Setting up local feed with unreleased Grpc.* packages:
+Activate the development environment:
```

Comment:
Can we be more descriptive? I don't know what activate means.

Is this needed if someone is using startvs.cmd?